WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -49,6 +49,10 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
         observeCardPresentPaymentEvents()
         observeItemsInCartForCartTotal()
     }
+    
+    var itemToScrollToWhenCartUpdated: CartItem? {
+        return itemsInCart.last
```

Comment:
I added this helper property to future proof it in case we start adding items to the cart to the beginning instead of to the end (as we do now)

Reply:
💭  As `PointOfSaleDashboardViewModel` is used for all POS views, it might be worth making sure this view model only has properties that are used across views. This property feels like it's specific to the cart view, WDYT if we start moving some properties from to `CartView`'s own view model like `CartViewModel`? `PointOfSaleDashboardViewModel.itemsInCart` can be passed as a publisher, and `PointOfSaleDashboardViewModel.orderStage` can be DI'ed as well.